api/src/extensions/lib/get-extensions-settings.ts

```diff
@@ -66,8 +99,12 @@ export const getExtensionsSettings = async ({
 	};
 
 	for (const [folder, extension] of local.entries()) {
-		const settingsExist = localSettings.some((settings) => settings.folder === folder);
-		if (settingsExist) continue;
+		const existingSettings = localSettings.find((settings) => settings.folder === folder);
+
+		if (existingSettings) {
```

Comment:
Place check here (as it's done for module and registry extensions)

```suggestion
		if (existingSettings) {
			if (extension.type === 'bundle') {
				updateBundleEntriesSettings(existingSettings, extension, localSettings);
			}

```